### PR TITLE
feat: add clim to AcquisitionParameter.plot_traces()

### PR DESCRIPTION
**Pull request by nulinspiratie**
Enables `AcquisitionParameter.plot_traces(clim=(0,1))

### CODE REVIEW

**Comment by maij** [comment]
Once [this](https://github.com/nulinspiratie/Qcodes/pull/89/files) is merged, we might want to add the `norm` option as well as clim.

File: silq/parameters/acquisition_parameters.py [hunk 1]
```diff
@@ -329,6 +329,7 @@ def analyse(self,
 
     def plot_traces(self, filter=None, channels=['output'],
                     t_skip: Union[bool, float] = True,
+                    clim=None,
```
Comment (RostyslavSavytskyy) [inline silq/parameters/acquisition_parameters.py:332]: general question: what is clim?

**Review by RostyslavSavytskyy — APPROVED** [review]

File: silq/parameters/acquisition_parameters.py (hunk 1 above)
Comment (nulinspiratie) [reply silq/parameters/acquisition_parameters.py:332]: haha you forced me to write documentation

Comment (RostyslavSavytskyy) [reply silq/parameters/acquisition_parameters.py:332]: I see, all good now

**Pull request merged**